### DISCUSSION TITLE
How to support PostgreSQL database upgrade with multiple tablespaces

### DISCUSSION THREAD

**Discussion by cobolbaby** [tianon/docker-postgres-upgrade#93] [General]
Currently, to support data tiered storage, I have configured multiple tablespaces. Does the pg_upgrade container support this configuration?

**Comment by tianon** [comment]
That's probably a better question for a proper postgres community; I actually don't know much about the utility other than I don't like it and the way it works requires an approach like this repo for containers. 😅
